tests/integration/BaseTest.cs

```diff
@@ -37,10 +48,26 @@ private static void TryLoadFromDotEnv(string fileName)
             }
         }
 
+        private void DecodeAccessKeyAndUpdateEnvVar()
+        {
+            var base64Encoded = Environment.GetEnvironmentVariable(AccessKeyVar);
+            if (base64Encoded != null)
+            {
+                var decoded = Encoding.UTF8.GetString(Convert.FromBase64String(base64Encoded));
+                Environment.SetEnvironmentVariable(AccessKeyVar, decoded);
+                HasBase64Decoded = true;
+            }
+        }
+
         private void PopulateFromEnv()
         {
-            ServicePrincipalKey = Environment.GetEnvironmentVariable("DEV_CA_PUBLIC_USE_TESTOAUTHSERVICEPRINCIPAL_SERVICE_PRINCIPAL_KEY");
-            AccessKey = JsonConvert.DeserializeObject<AccessKey>(Environment.GetEnvironmentVariable("DEV_CA_PUBLIC_USE_INTEGRATION_TEST_ACCESS_KEY"));
+            ServicePrincipalKey = Environment.GetEnvironmentVariable(SpKeyVar);
+            
+            if (!HasBase64Decoded)
```

Comment:
Could we just decode the base64 access key here instead? And then use it directly to deserialize it into the AccessKey object without saving it back to the env?

Reply:
Yeah, let me update the code.